src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,55 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public static int splitAndSum(String text) {
+        if (isBlank(text)) {
+            return 0;
+        }
+
+        return sum(toInt(split(text)));
+    }
+
+    private static boolean isBlank(String text) {
+        return text == null || text.isEmpty();
+    }
+
+    private static int[] toInt(String[] values) {
+        int[] numbers = new int[values.length];
+
+        for (int i = 0; i < values.length; i++) {
+            numbers[i] = toPositive(values[i]);
+        }
+
+        return numbers;
+    }
+
+    private static int toPositive(String value) {
+        int number = Integer.parseInt(value);
+        if (number < 0 ) {
```

Comment:
```suggestion
        if (number < 0) {
```
공백 컨벤션 위반입니다. IDE에서 컨벤션을 자동으로 맞춰주는 shortcut이 있으니 확인해보세요.
intelliJ 는 option + command + l 입니다.

8.2 blank spaces 항목을 확인해보세요.
https://myeonguni.tistory.com/1596

Reply:
아니 이런!  최종 커밋 전에 포맷팅 한다는 걸 깜빡했네요.🤪